packages/calling/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -114,12 +148,13 @@ describe('CallingClient Tests', () => {
     it('Verify invalid service indicator, empty domain', async () => {
       /* eslint-disable @typescript-eslint/no-explicit-any */
       const serviceDataObj: any = {indicator: 'test', domain: ''};
-
-      expect(() => {
-        callingClient = createClient(webex, {serviceData: serviceDataObj});
-      }).toThrow(Error);
-      expect(callingClient).toBe(undefined);
-      expect(createRegistrationSpy).not.toHaveBeenCalled();
+      try {
+        callingClient = await createClient(webex, {serviceData: serviceDataObj});
+      } catch (e) {
+        expect(e.message).toMatch(
+          'Invalid service indicator, Allowed values are: calling,contactcenter'
```

Comment:
```suggestion
        expect(e.message).toEqual(
          'Invalid service indicator, Allowed values are: calling,contactcenter'
```
Since we are using a full string match anyway, better to avoid `toMatch` and simply use `toEqual`.